plugins/chat/app/models/category_channel.rb

```diff
@@ -21,7 +21,26 @@ def title(_ = nil)
     name.presence || category.name
   end
 
-  def slug
-    title.truncate(100).parameterize
+  def ensure_slug
+    return if title.blank?
+    slug_title = self.title.strip
```

Comment:
I don't think we need those 2 lines here. We can assign slug_title when self.slug doesn't exist